project-base/k8s/ingress.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: shopsys
+spec:
+  rules:
+  - host: $HOSTNAME
```

Comment:
Please rename to $FIRST_DOMAIN_HOSTNAME and $SECOND_DOMAIN_HOSTNAME (as used above). The copy-pasted variable looks weird.